### DISCUSSION TITLE
feature request: add issue to current sprint with --current flag

### DISCUSSION THREAD

**Discussion by tomnieuwland** [ankitpokhrel/jira-cli#424] [Ideas]
G'day

First of all, thanks for the great tool. It's extremely useful.

I was wonder if it is possible to add an issue to the current sprint using something like this

`jira sprint add --current [...ISSUE-N]`

or alternatively even the ability to create a new issue in a sprint with something like

(where `jira sprint current` returns the ID for the current sprint)

`jira issue create --sprint $(jira sprint current)`

Once again, thanks for the tool!

**Comment by mochja** [comment]
I found a way to create an issue with current sprint.

```
jira issue create --custom sprint=$(jira sprint list --table --plain --columns=id --state=active | awk 'NR==2')
```

Also see the [caveats section](https://github.com/ankitpokhrel/jira-cli/discussions/346), the custom sprint field may be wrongly defined.

**Comment by php-coder** [comment]
@mochja BTW, the command `jira sprint list --table --plain --columns=id --state=active | awk 'NR==2'`  can be simplified into `jira sprint list --table --plain --columns=id --state=active --no-headers`

**Comment by joaonc** [comment]
I would love this feature also!  And thanks for the workarounds on retrieving the current sprint ID.
